client/components/map/mapToolTip.jsx

```diff
@@ -1,35 +1,46 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { Popup } from 'react-leaflet'
-const Tooltip = ({distance, trailName, isActive, ...props}) => (
-  <Popup {...props}>
-    <span className='trail__tooltip'>
-      <h5>
-        { trailName || 'No name given' }
-      </h5>
-      <p>Estimated Distance: {isActive 
-      ? guessLength(props.popupContainer, distance.addLength) + 'Miles'
-      : '' 
-      }
-      </p>
-    </span>
-  </Popup>
-)
 
-function guessLength(layer, addLength) { //get the id out of the layer?
-  let tempLatLng = null
-  const { _latlngs, feature } = layer._layers[layer._leaflet_id-1]
-  const distance = _latlngs.reduce((totalDistance, latlng)=> {
-    if(tempLatLng == null){
+export default class Tooltip extends Component {
```

Comment:
Is there a reason for the shift to extending Component instead of just the `const` approach?